### DISCUSSION TITLE
Predetermination of relevant parameters for QCxMS (x = EI or CID) simulations

### DISCUSSION THREAD

**Discussion by ghost** [qcxms/QCxMS#68] [General]
As stated in the documentation (https://xtb-docs.readthedocs.io/en/latest/qcxms_doc/qcxms_run.html#ei-method-specific-keywords and https://xtb-docs.readthedocs.io/en/latest/qcxms_doc/qcxms_run.html#cid-method-specific-keywords), there are parameters that have significant influence on the fragmentation pattern of the molecular ion and therefore, the resulting mass spectrum.  I reckon that
For EI, the important set of parameters is `[eimp0, eimpw, ieeatm, tmax]`
For CID (for `fullauto` mode),  the important set of parameters is `[elab, pgas, lchamb, tmax]`
Can certain simulations be run to obtain a (good) initial guess of these sets ?

Furthermore, I have the following questions:
1) Is the value for `eimpw` the most probable impact excess energy per atom of the corresponding distribution ? 
2) How will I know if the simulation fails because of the type of IEE distribution ? 
3) How should the limits for the distribution be set ?

**Comment by JayTheDog** [comment]
Overall, the **default setting** that are listed in the docs that you have provided are those that were found to be the best initial guess for *most* systems. This refers to a majority of molecules, but does not mean that it is the best for particular cases. This means you **do not have to figure out all of the setting by your own**, the defaults will take care of most cases!

If you want to customize your calculations, it *depends* on your system to which you are comparing. Take a look at the molecular ion and change the values correspondingly. Too large M+ / [M+H]+ peaks means you have to increase the internal energy.
- For CID, use `fullauto` and adapt the `elab` to find the best value. 
- For EI, you could increase `eimp0` and `ieeatm` (the latter in small steps).

For **CID**, the nature of the instruments that are used have a significant influence on the outcome, so its not possible to find a overall *best* initial guess for the set. The `fullauto` method tries to calculate the number of of collisions dependend on `pgas` and `lchamb`, but you can also set these manually in the `collauto` runmode with `setcoll`. 

For the other questions:
1.  For `eimpw`, the default is the best value. This is the energy distribution *width*, meaning a Gaussian distribution at the energy impact site. This value is not that significant, wouldn't change.
2. The computation should not fail because of the *type* of IEE distribution, you are only getting different kinds of results, i.e. signal intensities. Poisson showed better results (thus it's the default). Wouldn't change either.
3. Read the literature https://onlinelibrary.wiley.com/doi/10.1002/anie.201300158 (especially the supporting information)
